pkg/metrics/server.go

```diff
@@ -422,6 +426,20 @@ func InitServerMetrics() {
 			Name:      "active_users",
 			Help:      "The total count of active user.",
 		})
+	TLSVersion = metricscommon.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "server",
+			Name:      "tls_version",
+			Help:      "Counter per TLS Version.",
+		}, []string{LblVersion})
+	TLSCipher = metricscommon.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "server",
+			Name:      "tls_cipher",
+			Help:      "Counter per TLS Cipher.",
+		}, []string{LblCipher})
```

Comment:
Are LblCipher and LblVersion enumerable? I ask this question because metrics vec with high distinct values will cost much more resource.

Reply:
For `LblVersion`: there are two possible values: `TLSv1.2` and `TLSv1.3`. See `versionString` in `pkg/util/tls/tls.go`.

For `LblCipher`: there are 25 values. See `tlsCipherString` in `pkg/util/tls/tls.go`. However in a production setup I would expect something like 5 distinct values.